### ISSUE TITLE
Query more than 250 products pro collection

### ISSUE THREAD

**Issue by chrispreisler** [thetre97/gridsome-source-shopify#37]
Hey,

is it possible to query for more than 250 products pro collection. I have a rather large collections with around 1000 products. I saw in your source plugin, that you limit the products to 250. I thought I can maybe overwrite this in my page-query, but unfortunately that doesn't work.

```
<page-query>
query Collection ($id: ID!) {
  shopifyCollection (id: $id) {
    id
    title
    description
    products (limit: 500) {
      id
      title 
    }
  }
}
</page-query>
``` 

Thank you very much for your help.
Best regards
Chris

**Comment by chrispreisler** [comment]
When I query only for the products and set the limit to 1000, it is working as expected and I get all products. Is it possible to integrate the bulk load you use in your code with queryAll also for the products related to one collection?

**Comment by travis-r6s** [comment]
@chrispreisler Have just published a new version `0.1.13` - could you check to see if that lists all products in a collection?

**Comment by chrispreisler** [comment]
Thank you very much, works perfect.
